AssemblerLib/LocalToGlobalIndexMap.h

```diff
@@ -31,32 +35,31 @@ namespace AssemblerLib
 class LocalToGlobalIndexMap
 {
 public:
+#ifdef USE_PETSC
+    typedef MathLib::RowColumnIndices<PetscInt> RowColumnIndices;
```

Comment:
Tried. If this ``MathLib::RowColumnIndices<PetscInt> RowColumnIndices`` is moved, I have to replace PetscInt with std::size_t in the related arguments of all member functions of PETScMatrix and PETScVector. I would like to keep this for flexibility.  If it is definitely needed to remove this, I will do it in next PR, where the changes are only to remove ifdef and modify  PETScMatrix and PETScVector.